cpu/atmega1281/include/periph_cpu.h

```diff
@@ -42,6 +42,13 @@ enum {
     PORT_G = 6,       /**< port G */
 };
 
+/**
+ * @brief   Defines for the I2C interface
```

Comment:
while here its the opposite, change `@brief` to `@name`
and add group opening `@{`